src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/StructuralVariantContext.java

```diff
@@ -82,13 +94,14 @@ private StructuralVariantContext(final VariantContext other) {
      */
     @Override
     public int getEnd() {
-        if (end <= 0) {
-            if (hasAttribute(VCFConstants.END_KEY)) {
-                end = getAttributeAsInt(VCFConstants.END_KEY, -1);
-            } else if (getStructuralVariantType() == StructuralVariantType.DEL) {
-                end = getStart() + getStructuralVariantLength();
-            } else {
-                end = super.getEnd();
+        if (end == MISSING_END) {
```

Comment:
Not suggesting that you change them now (though it will be good if you do so), I still believe that BND records should have no `END` concept associated, so having a `hasEnd()` method is good, and preferably users should call `hasEnd()` first, then `getEnd()`.
But as said, you don't need to add this now.